src/firmware/shared/circular_buffer.h

```diff
@@ -0,0 +1,74 @@
+#include <stdbool.h>
+#include <stdint.h>
```

Comment:
nit:not sure if this is still needed

Reply:
`stdint.h` is not needed but `stdbool.h` will throw errors of "Unknow type name 'bool'."

Reply:
sry yeah I meant stdint

Reply:
Done.